src/PrestaShopBundle/Resources/config/routing.yml

```diff
@@ -6,3 +6,8 @@ _admin_routing:
 _api_routing:
   resource: "@PrestaShopBundle/Resources/config/routing/api.yml"
   prefix: /api
+
+_new_api_routing:
+  resource: .
+  type: api_platform
+  prefix: /new-api
```

Comment:
Can we have a real path like `/v2/` instead of `/new-api` ?

Reply:
Yeah 😄, I know but we didn't really take the decision about the api versioning yet. Also this route will eventually be deleted as it exists only to have an endpoint for testing purposes while the "real" API is being developped.